validator/sql_location_test.rego

```diff
@@ -0,0 +1,165 @@
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GCPSQLLocationConstraintV1
+
+import data.test.fixtures.sql_location.assets.sql_instances as fixture_sql_instances
+import data.test.fixtures.sql_location.constraints as fixture_constraints
+
+find_violations[violation] {
+	asset := data.assets[_]
+	constraint := data.test_constraints[_]
+	issues := deny with input.asset as asset with input.constraint as constraint
+	total_issues := count(issues)
```

Comment:
delete this since `total_issues` doesn't seem to be used?

Reply:
removed